modules/openfast-library/src/FAST_Types.f90

```diff
@@ -6895,17 +6895,17 @@ subroutine FAST_PackOutputFileType(Buf, Indata)
    type(PackBuffer), intent(inout) :: Buf
    type(FAST_OutputFileType), intent(in) :: InData
    character(*), parameter         :: RoutineName = 'FAST_PackOutputFileType'
-   integer(IntKi)  :: i1, i2
-   integer(IntKi)  :: LB(2), UB(2)
+   integer(B8Ki)   :: i1, i2
+   integer(B8Ki)   :: LB(2), UB(2)
    if (Buf%ErrStat >= AbortErrLev) return
    call RegPack(Buf, allocated(InData%TimeData))
    if (allocated(InData%TimeData)) then
-      call RegPackBounds(Buf, 1, lbound(InData%TimeData), ubound(InData%TimeData))
+      call RegPackBounds(Buf, 1, lbound(InData%TimeData, kind=B8Ki), ubound(InData%TimeData, kind=B8Ki))
       call RegPack(Buf, InData%TimeData)
    end if
    call RegPack(Buf, allocated(InData%AllOutData))
    if (allocated(InData%AllOutData)) then
-      call RegPackBounds(Buf, 2, lbound(InData%AllOutData), ubound(InData%AllOutData))
+      call RegPackBounds(Buf, 2, lbound(InData%AllOutData, kind=B8Ki), ubound(InData%AllOutData, kind=B8Ki))
```

Comment:
This new registry looks a bit different than the old one, but I think this is the spot we had issues in the old one.